Dockerfile.distroless

```diff
@@ -0,0 +1,43 @@
+FROM golang:1.17-buster as builder
```

Comment:
This should be `FROM golang:1.17-bullseye as builder`

Reply:
Updated to `golang:1.17-bullseye`